boards/tsi/tsi/board.yml

```diff
@@ -1,10 +1,9 @@
 board:
-  name: ek_tsi
+  name: tsi
   vendor: Tsavorite
   socs:
   - name: skyp
     variants:
-    - name: m85
+    - name: cortex-m85
       cpucluster: m85
-    # - name: xtensa
-    #   cpucluster: xtensa
+  - name: mimx8ml8
```

Comment:
If you remove this mim8ml8 and just make it txe, in this file board/tsi/tsi/board.yml file you can remove the requirement of this being called imx8ml8 so you can just call it txe without adsp and 8imx8ml8 extension requirement.

Reply:
resolving this comment as I made changes to address this, the new target is "tsi/skyp/txe" now for TXE.